lib/griddler/version.rb

```diff
@@ -1,3 +1,3 @@
 module Griddler
-  VERSION = '1.4.0'
+  VERSION = '1.4.1'
```

Comment:
Can this change be removed please? In general, it's better to let the library maintainers worry about version bumps and such. Also, as this is a backwards compatible API change, this will end up being a minor release vs a patch release.